src/app/helptext/sharing/smb/smb.ts

```diff
@@ -89,11 +89,8 @@ export const helptext_sharing_smb = {
  target="blank">virtual file system modules</a> to\
  enhance functionality.'),
 
-    placeholder_storage_task: T('Periodic Snapshot Task'),
-    tooltip_storage_task: T('Used to configure directory shadow copies on a\
- per-share basis. Select the pre-configured periodic\
- snapshot task to use for the shadow copies of this\
- share. Periodic snapshots must be recursive.'),
+    placeholder_shadowcopy: T('Enable Shadow Copies'),
+    tooltip_shadowcopy: T('Export ZFS snapshots as Shadow Copies for VSS clients.'),
```

Comment:
Make "Shadow Copies" in the help text a link to: https://docs.microsoft.com/en-us/windows/desktop/vss/shadow-copies-and-shadow-copy-sets